### PR TITLE
LPD-42687 Can Download Client Extensions 

### PR DESCRIPTION
**Pull request by CalebJamesH**
https://liferay.atlassian.net/jira/software/c/projects/LPD/boards/256?quickFilter=4481&selectedIssue=LPD-42687

### CODE REVIEW

**Comment by CalebJamesH** [comment]
ci:test:sf

**Comment by CalebJamesH** [comment]
ci:test:sf

File: workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-custom-element/src/enums/ProductSpecification.ts [hunk 1]
```diff
@@ -6,3 +6,13 @@
...
+
+export enum SpecificationKey {
+	DEVELOPER_NAME = 'developer-name',
+	DOWNLOADABLE_CLOUD = 'downloadable-cloud',
+	LATEST_VERSION = 'latest-version',
+	LICENSE_TYPE = 'license-type',
+	LIFERAY_VERSION = 'liferay-version',
+	PRICE_MODEL = 'price-model',
+	TYPE = 'type',
+}
```
Comment (kevenleone) [inline workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-custom-element/src/enums/ProductSpecification.ts:18]: Already exists `PRODUCT_SPECIFICATION_KEY`

src/enums/Product.ts

```suggestion
```

File: workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-custom-element/src/pages/CustomerDashboard/pages/Apps/App/AppOutlet.tsx [hunk 2]
```diff
@@ -113,13 +114,33 @@ const AppOutlet = () => {
...
+									path: 'cloud-provisioning',
+								},
+							]
+						: [
+								...tabs,
+								{
+									name: i18n.translate('app-provisioning'),
+									path: 'cloud-provisioning',
+								},
+							];
```
Comment (kevenleone) [inline workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-custom-element/src/pages/CustomerDashboard/pages/Apps/App/AppOutlet.tsx:143]:
```suggestion
					return [
								...tabs,
								{
									name: i18n.translate('download'),
									path: 'download',
									visible: isCompletedOrderWithVirtualItems && isDownloadableCloud,
								},
								{
									name: i18n.translate('app-provisioning'),
									path: 'cloud-provisioning',
								},
							]
```

**Review by kevenleone — CHANGES_REQUESTED** [review]

**Comment by kevenleone** [comment]
ci:test:relevant

**Comment by kevenleone** [comment]
ci:forward:force

**Pull request closed without merging**